test/attr/attr_override.swift

```diff
@@ -40,7 +40,7 @@ class A {
   var v9: Int { return 5 } // expected-note{{attempt to override property here}}
   var v10: Int { return 5 } // expected-note{{attempt to override property here}}
 
-  subscript (i: Int) -> String { // expected-note{{potential overridden subscript 'subscript(_:)' here}}
+  subscript (i: Int) -> String { // expected-note{{potential overridden subscript 'subscript(_:)' here}} expected-note{{potential overridden subscript 'subscript(_:)' here}}
```

Comment:
Hm, shouldn't we only be showing overridables with the same static-ness?

Reply:
You're right—we don't show that for class functions (although I'm not actually sure where we test that). I'll look into this.

Reply:
No, it looks like I was right. This note was generated by the new instance subscript on line 181, which was added to test that the compiler doesn't think an instance subscript can override a class subscript with the same signature. But I could avoid these notes by tweaking the test a little.

Reply:
Improving that test revealed [a subscript override-checking bug unrelated to static subscripts](https://bugs.swift.org/browse/SR-10323). I've pushed the modified test with FIXMEs referring to this bug, but I'll amend it if I end up fixing the new bug first.